cob_frame_tracker/src/cob_frame_tracker.cpp

```diff
@@ -141,7 +141,7 @@ bool CobFrameTracker::initialize()
     lookat_focus_frame_ = "lookat_focus_frame";
 
     // ABORTION CRITERIA:
-    enable_abortion_checking_ = true;
+    enable_abortion_checking_ = false;
```

Comment:
leave the default value to `true`...as it safer...
this parameter is configurable via parameter yaml and dynamic_reconfigure anyway